src/gui/ipsubnetwhitelistoptions.cpp

```diff
@@ -37,6 +34,8 @@
 
 #include "base/preferences.h"
 #include "base/utils/net.h"
+#include "ipsubnetwhitelistoptions.h"
```

Comment:
"Module" header should be the first, then system and Qt headers, then all other application headers (according to Coding Guidelines).